share/scripts/inverse/run_gromacs.sh

```diff
@@ -115,12 +117,32 @@ if [[ ${CSG_MDRUN_OPTS} ]]; then
   mdrun_opts+=" ${CSG_MDRUN_OPTS}"
 fi
 
+#support of REMD during gromacs simlation
+if [[ $(csg_get_property cg.inverse.gromacs.REMD) = "yes" ]]; then
+PREFIX="$(csg_get_property cg.inverse.gromacs.REMD.PREFIX)"
+read -a T <<<"$(csg_get_property cg.inverse.gromacs.REMD.T_for_replicas)"
+j=0
+for i in "${T[@]}"
+do
+echo $i
+#TEMP has to be written as a place holer for the temperature in the *.mdp file
+sed "s/TEMP/$i/" grompp.mdp > grompp_$j.mdp
+critical ${grompp[@]} -n "${index}" -f "grompp_$j.mdp" -p "$topol_in" -o "$PREFIX"_"$j.tpr" -c "${conf}" ${grompp_opts} 2>&1 | gromacs_log "${grompp[@]} -n "${index}" -f "${mdp}" -p "$topol_in" -o "$tpr" -c "${conf}" ${grompp_opts}"
```

Comment:
Hi Andrey, thanks for the comment. Let me quickly reply:

To 2.: Prefix is also used for the output trajectories. And I basically followed the explanations and syntax of gromacs. (see http://www.gromacs.org/Documentation/How-tos/REMD)

To 3: I like your suggestion

To 4: Can you specify what do you mean by replacing 'ref-t' ? Do you mean to leave the right hand of 'ref t' blank or deleting the whole line in the .mdp file and parse it within the run_gromacs script ? 

Cheers,
David